### ISSUE TITLE
Questions about dataset partitioning

### ISSUE THREAD

**Issue by TonyZhang2022** [aravindsankar28/DySAT#28]
Thank you so much for this great job! It helped me a lot.

What I want to know is why is the test set 60% and the training set only 20%.
This is also the case in the original paper.

Looking forward to your reply, thank you again.
